src/qt/walletframe.cpp

```diff
@@ -87,9 +87,8 @@ bool WalletFrame::removeWallet(const QString &name)
 
 void WalletFrame::removeAllWallets()
 {
-    QMap<QString, WalletView*>::const_iterator i;
-    for (i = mapWalletViews.constBegin(); i != mapWalletViews.constEnd(); ++i)
-        walletStack->removeWidget(i.value());
+    for (const auto& i : mapWalletViews)
+        walletStack->removeWidget(i);
     mapWalletViews.clear();
```

Comment:
These changes (except for the last one that is actually about names) are unrelated to your change.  While I in general think they are good (using C++11 for-each loops), there is also value in keeping code unchanged wrt upstream Bitcoin - it may avoid future merge conflicts.  So I would personally not make these changes, and only use a C++11 loop in the new code you are adding.  But if you prefer to do this change at the risk of merge conflicts later on, that's also fine with me.